src/common/loadingBar.js

```diff
@@ -0,0 +1,52 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+import Bar from 'react-native-progress/Bar';
+import { getLoading } from '../selectors';
+import { BRAND_COLOR, NAVBAR_SIZE } from '../styles';
+import { connect } from '../react-redux';
+import type { Dispatch } from '../types';
+
+type SelectorProps = $ReadOnly<{|
+  loading: boolean,
+|}>;
+
+type Props = $ReadOnly<{|
+  color: string | number,
+  unfilledColor: string,
+
+  dispatch: Dispatch,
+  ...SelectorProps,
+|}>;
+
+class LoadingBar extends PureComponent<Props> {
+  static defaultProps = {
+    color: BRAND_COLOR,
+    unfilledColor: 'white',
+  };
+
+  render() {
+    const { loading, ...rests } = this.props;
+
+    return (
+      <View style={{ position: 'absolute', left: 0, right: 0, top: NAVBAR_SIZE - 3 }}>
```

Comment:
Can you make a local variable `const height = 3`, and use that here and to set the height below? that way we can avoid this magic number